src/shared/record/record.ts

```diff
@@ -98,6 +131,18 @@ export class Record {
           .filter(Story.isProtoStory)
           .map(Story.fromObj));
     }
+    if ('images' in obj && obj.images instanceof Array) {
+      record.addImages(
```

Comment:
I'd do this in the addImages method, not here.

I think I'll move the story dedupe that way, as well.